static/lib/material-icons/material-icons.css

```
@@ -1 +1 @@
-@font-face{font-display:swap;font-family:Material Icons;font-style:normal;font-weight:400;src:url(../../lib/material-icons/v77/icons.woff2) format("woff2")}.material-icons{display:inline-block;font-family:Material Icons;font-size:24px;font-style:normal;font-weight:400;letter-spacing:normal;line-height:1;text-transform:none;white-space:nowrap;word-wrap:normal;direction:ltr;-webkit-font-feature-settings:"liga";-webkit-font-smoothing:antialiased}
+@font-face{font-display:swap;font-family:Material Icons;font-style:normal;font-weight:400;src:url(../../lib/material-icons/v145/icons.woff2) format("woff2")}.material-icons{display:inline-block;font-family:Material Icons;font-size:24px;font-style:normal;font-weight:400;letter-spacing:normal;line-height:1;text-transform:none;white-space:nowrap;word-wrap:normal;direction:ltr;-webkit-font-feature-settings:"liga";-webkit-font-smoothing:antialiased}
```

Stylelint (17.11.0)

[error] Expected quotes around "Material Icons" (font-family-name-quotes)

(font-family-name-quotes)